src/reducers/versions.tsx

```diff
@@ -974,6 +1087,71 @@ export const fetchVersionFile = ({
   };
 };
 
+type FetchDiffFileParams = {
+  _getDiffFileOnly?: typeof getDiffFileOnly;
+  addonId: number;
+  baseVersionId: number;
+  headVersionId: number;
+  path: string;
+};
+
+export const fetchDiffFile = ({
+  _getDiffFileOnly = getDiffFileOnly,
```

Comment:
This line is missing coverage because the only tests we have right now are for the reducer itself, and we always pass a mock in for this. I believe when the next patch comes out, that integrates all the pieces together, we should have a test for the `Compare` component that provides coverage for this line.